extensions/amp-mega-menu/amp-mega-menu.md

```diff
@@ -0,0 +1,325 @@
+
+The following example demonstrates how to hide the mega menu when viewport width is no more than `500px` and instead show a button to open the sidebar containing an accordion menu.
+
+[example playground="true" preview="top-frame"]
+```html
+<head>
+  <script async custom-element="amp-mega-menu" src="https://cdn.ampproject.org/v0/amp-mega-menu-0.1.js"></script>
+  <script async custom-element="amp-sidebar" src="https://cdn.ampproject.org/v0/amp-sidebar-0.1.js"></script>
+  <script async custom-element="amp-accordion" src="https://cdn.ampproject.org/v0/amp-accordion-0.1.js"></script>
+  <style amp-custom>
+    .sidebar-open-btn {
+      font-size: 2em;
+      display: none;
+    }
+    @media (max-width: 500px) {
+      #mega-menu {
+        display: none;
+      }
+      .sidebar-open-btn {
+        display: block;
+      }
+    }
+  </style>
+</head>
+<body>
+  <header>
+    <amp-mega-menu id="mega-menu" height="50" layout="fixed-height">
+      <nav>
+        <ul>
+          <!-- list of menu items here -->
+          <li>
+            <h4 role="button">menu item</h4>
+            <div role="dialog">more content</div>
+          </li>
+        </ul>
+      </nav>
+    </amp-mega-menu>
+    <button class="sidebar-open-btn" on="tap:sidebar">=</button>
+  </header>
+  <amp-sidebar id="sidebar" layout="nodisplay">
+    <amp-accordion>
+      <!-- list of menu items here -->
+      <section>
+        <h4>menu item</h4>
+        <div>more content</div>
+      </section>
+    </amp-accordion>
+  </amp-sidebar>
+</body>
+```
+[/example]
+
+## Attributes
+
+<table>
+  <tr>
+    <td width="40%"><strong>data-close-button-aria-label (optional)</strong></td>
+    <td>Optional attribute used to set ARIA label for the close buttons added for accessibility.</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>common attributes</strong></td>
+    <td>This element includes <a href="https://amp.dev/documentation/guides-and-tutorials/learn/common_attributes">common attributes</a> extended to AMP components.</td>
+  </tr>
+</table>
+
+## Styling
+
+The `<amp-mega-menu>` component can be styled with standard CSS.
+
+- By default, a white background is applied to the `<nav>` element and the expandable content elements.
+- When open, the content containers will fill the entire viewport width but can be overridden with the `left` and `width` properties.
+- The current state of the mega menu is exposed via the `open` attribute that is set on the `amp-mega-menu` tag when one of the menu items is expanded.
+- The `<li>` element with expanded content will also receive the `open` attribute, which developers can use to style its children.
+
+In the example below, we will customize the background color of the navigation bar, the appearance of menu buttons when opened, as well as the positioning of dropdown containers.
```

Comment:
Clarify to:
The example below customizes:
- The background color the navigation bar
- The appearance of opened menu buttons
- The position of the dropdown containers